frontend/client_sdk/src/CodaSDK.d.ts

```diff
@@ -117,3 +117,10 @@ export declare const verifyStakeDelegationSignature: (signedStakeDelegation: sig
   * @returns A string that represents the JSON conversion of a signed Rosetta transaction`.
    */
 export declare const signedRosettaTransactionToSignedCommand: (signedRosettaTxn: string) => string;
+/**
+* Return the hex-encoded format of a public key. This will raise an exception
```

Comment:
This will not raise an exception now right?

Reply:
Yup, that's correct. I fixed the comments to be more clear.